docs/origin_install.md

```diff
@@ -0,0 +1,154 @@
+    hierarchy which you wish to serve via XRootD.
+
+        set rootdir = <DIRECTORY>
+
+    !!! danger "Carefully consider your `rootdir`"
+        Do not set `rootdir` to `/`.
+        This might result in serving private information.
+
+1.  If you want to limit the sub-directories to serve under your configured `rootdir`,
+    comment out the `all.export /` directive in
+    `/etc/xrootd/config.d/90-osg-standalone-paths.cfg`,
+    and add an `all.export` directive for each directory under `rootdir` that you wish to serve via XRootD.
+
+    This is useful if you have a mixture of files under your `rootdir`, for example from multiple users,
+    but only want to expose a subset of them to the world.
+
+    For example, to serve the contents of `/data/store` and `/data/public` (with `rootdir` configured to `/data`):
+
+        all.export /store/
+        all.export /public/
+
+    If you want to serve everything under your configured `rootdir`, you don't have to change anything.
+
+    !!! danger
+        The directories specified this way are writable by default.
+        Access controls should be managed via [authorization configuration](#configuring-authentication-and-authorization).
+
+1. In `/etc/xrootd/config.d/10-common-site-local.cfg`, add a line to set the `resourcename` variable.
+   Unless your supported VOs' policies state otherwise,
+   this should match the [resource name](../../common/registration.md#registering-resources) of your XRootD service.
+   For example, the XRootD service registered at the
+   [University of Florida site](https://github.com/opensciencegrid/topology/blob/b14218d6e9d9df013a42e4d8538b2eeea615514c/topology/University%20of%20Florida/UF%20HPC/UFlorida-HPC.yaml#L250)
+   should set the following configuration:
+
+        set resourcename = UFlorida-XRD
+
+   For example, the XRootD service registered at the
+   [University of Florida site](https://github.com/opensciencegrid/topology/blob/b14218d6e9d9df013a42e4d8538b2eeea615514c/topology/University%20of%20Florida/UF%20HPC/UFlorida-HPC.yaml#L250)
+   should set the following configuration:
+
+        set resourcename = UFlorida-XRD
+
+### Configuring authentication and authorization
+
+XRootD offers several authentication options using security plugins to validate incoming credentials, such as bearer tokens, X.509 proxies, and VOMS proxies.
+Please follow the [XRootD authorization documentation](xrootd-authorization.md) for instructions on how to configure authentication and authorization,
+including validating credentials and mapping them to users if desired.
+
+### Optional configuration
+
+The following configuration steps are optional and will likely not be required for setting up a small site.
+If you do not need any of the following special configurations, skip to
+[the section on using XRootD](#using-xrootd).
+
+
+#### Enabling multi-user support
+
+!!! warning "Requirements for XRootD-Multiuser with VOMS FQANs"
```

Comment:
I don't know much about markdown so I could be wrong, but maybe reformat these warnings to look more visually appealing. Could just be github not recognizing '!!!' or something too.

Reply:
Hmm. I can't see a good way to do this. At least in the markdown nextra uses